MozillaFoundation/foundation.mozilla.org
# Character counter on page edit now adjusts text color based on selected admin theme

# Description

This PR removes a style override to the .max-length-contdown class in the wagtail admin.  This will allow the character countdown text color to change based on different themes.

Link to sample test page: https://foundation-s-11809-char-kgrsmr.herokuapp.com/cms/pages/13/edit/
Credentials: admin / 0g^1N2wWcl%wxZIY
Related PRs/issues: #11809 

# Steps to test

1. Log into the wagtail admin
2. Go to Account and select an admin theme
3. Edit a Blog page (see link to sample test page for example)
4. Verify the page counter is light if a dark theme is set, or dark if a light theme is set.
5. Repeat steps 2-4 for each admin theme.
